.gitignore

```diff
@@ -4,3 +4,4 @@ dist/
 browserstack.err
 local.log
 *.tgz
+.envrc
```

Comment:
Preferably put this into your global `~/.gitignore` instead since this is not project-specific but specific to the fact that you are using direnv.

Reply:
done